src/engine/level_script.c

```diff
@@ -735,6 +736,41 @@ static void level_cmd_show_dialog(void) {
     sCurrentCmd = CMD_NEXT;
 }
 
+#ifdef BETTER_REVERB
```

Comment:
this seems like code duplication kinda, would it be that messy to have these ifdefs inside the functions? i understand wanting to avoid lots of ifdef interleaving but having separate copies of the functions isn't much more friendly in cases like these, depends on how complicated they'd look with it

Reply:
The first two commands are different enough that I decided against scattering ifdefs around the functions, and largely justified it with the fact that VSCode darkens unused code anyway so it's just one giant block that can be ignored.

Reply:
but i just checked , they're barely different at all, they're both just one extra bit of code that could be neatly wrapped in one ifdef?

Reply:
Each function only needs one ifdef but the first one would need an else with it. If you still think that's better then I'll change it to do that instead.

Reply:
oh, i see now. would it be possible to instead rearrange the command params so that music param 2 stays at index 4?

Reply:
I think so, but it would either require a new `CMB_HBB` macro or some bitwise operations in the level script macro.

Reply:
personally i'm fine with a new command like that since it's general use and good for other custom commands, but maybe better to see what others think instead of adding it on a whim to simplify an ifdef

Reply:
Actually wait a minute, I can just make both musicParam2 offsets use index 6. Problem solved.